classes/controller/Controller.php

```diff
@@ -192,32 +193,14 @@ public static function getController($class_name, $auth = false, $ssl = false)
 
     public function __construct()
     {
-        if (is_null($this->display_header)) {
```

Comment:
I'm not sure of the change you introduce here, because I can't even imagine what the developer wanted to do when he introduced the checks? 

But let's say this was used to set a value for these parameters if they were not defined in children classes:

```php
class ChildrenController extends Controller
{
    public function __construct()
    {
        $this->display_header = false;

        return parent::__construct(); // really common call ^^
    }
}
````

Before your update: ``$this->display_header`` equals ``false``

After your update: ``$this->display_header`` equals ``true``

This is why we shouldn't refactor this specific part, even if this has no impact on Core, but I'll ask review from @Quetzacoalt91 as I'm not really aware of what does this code in the end

Reply:
@mickaelandrieu I did not think about this case. To keep it simple I just reintroduced the is-null-checks. Thank you for the review!